packages/eas-cli/src/commands/branch/publish.ts

```diff
@@ -37,11 +42,21 @@ export default class BranchPublish extends Command {
       description: `return a json with the edited branch's ID and name.`,
       default: false,
     }),
+    'custom-bundler': flags.boolean({
```

Comment:
`--custom-bundler` sounds like something that takes a name / command as an argument. Maybe `--skip-bundle` or `--skip-export` would be more self descriptive?

Reply:
Or perhaps don't have this flag, and skip bundling if `--input-dir` is given. This would make it very clear that you're either going to use expo-cli or provide a path to an already bundled app. It would also make it explicit where we look for the custom bundled output.

Reply:
the `--skip-bundler` command makes sense to me.

I think we should avoid overloading the `--input-dir`. It seems OK to allow the developer to both specify a custom input dir *and* have us bundle things automatically.